contracts/amm.cairo

```diff
@@ -200,7 +210,8 @@ func close_position{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_chec
     option_size : Int, // in base token
     quote_token_address: Address,
     base_token_address: Address,
-    lptoken_address: Address
+    lptoken_address: Address,
+    limit_desired_price: Math64x61_, // Should be total premia including fees - w.r.t. to opposite side
```

Comment:
This is called a price but it's not per option, I'd call this limit_premia

Reply:
done